src/server/game/Globals/WorldGlobals.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of the AzerothCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by the
+ * Free Software Foundation; either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "Log.h"
+#include "QueryResult.h"
+#include "Timer.h"
+#include "WorldGlobals.h"
+
+WorldGlobals* WorldGlobals::instance()
+{
+    static WorldGlobals instance;
+    return &instance;
+}
+
+void WorldGlobals::LoadAntiDosOpcodePolicies()
+{
+    uint32 oldMSTime = getMSTime();
+
+    _antiDosOpcodePolicies.fill(nullptr);
```

Comment:
Would _antiDosOpcodePolicies = {} work here to just reinitialize it?

Reply:
Yeah, I forgot it was a unique_ptr. I guess I deserve that one for committing just before falling asleep.